src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -984,15 +984,16 @@ public synchronized Response changeLiveVersion(@Context final HttpServletRequest
 
         String oldLiveVersion = contentManager.getCurrentContentSHA();
 
-        HttpClient httpClient = new DefaultHttpClient();
+        HttpResponse httpResponse;
+        try (CloseableHttpClient httpClient = HttpClientBuilder.create().build()) {
+          HttpPost httpPost = new HttpPost("http://" + getProperties().getProperty("ETL_HOSTNAME") + ":"
```

Comment:
Would it be worth extracting this get-properties-and-format section in the same way as has been done in the InfoFacade?

Reply:
Good idea, it's more readable that way 👍